aten/src/ATen/native/mkldnn/View.cpp

```diff
@@ -0,0 +1,37 @@
+#include <ATen/ATen.h>
+#include <ATen/Config.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/InferSize.h>
+
+#if !AT_MKLDNN_ENABLED()
+
+namespace at {
+namespace native {
+
+Tensor mkldnn_view(const Tensor& self, IntArrayRef size) {
+  AT_ERROR("mkldnn_view: ATen not compiled with MKLDNN support");
+}
+
+} // namespace native
+} // namespace at
+
+#else // AT_MKLDNN_EBABLED
+
+#include <ATen/native/mkldnn/MKLDNNCommon.h>
+
+namespace at {
+namespace native {
+
+Tensor mkldnn_view(const Tensor& self, IntArrayRef size) {
+  auto inferred_size = at::infer_size(size, self.numel());
+  const ideep::tensor& x = itensor_from_mkldnn(self);
+  ideep::tensor y;
+  ideep::direct_copy::compute<AllocForMKLDNN>(x, y);
+  y.reshape({inferred_size.cbegin(), inferred_size.cend()});
```

Comment:
this looks like a copy, not a view.

Reply:
yes it doesn't, but for opaque tensor I thought it's expected?